claw/MeasureCl.py

```diff
@@ -37,18 +38,24 @@ def _getbinlist(self):
                 lar.append(l)
         ## now get mapping from lmax to bins
         self.binlist=self.Cl.ndx[lar]
+        self.initbinlist=self.binlist
 
-    def _getIM(self,mp, addN=False):
+    def _getIM(self,mp, addN=False, m0neg=False):
         if addN:
             mp+=np.random.normal(0,self.Noise)
         mp*=self.Weight
         almsq=abs(hp.map2alm(mp)**2)
+        self.binlist=self.initbinlist
```

Comment:
To je treba popravit:

a) getIM je klican fulkrat, v vsakem primeru dajte binlist izracunat enkrat, ne zmerom znova, torej verjetno v initiu.
b) mislim, da je tudi narobe, ker morate almsq popravit... Z zdaj m=0 nacinov ne odrezete iz map2alm, ne?
c) preimenuje m0neg v ignorem0, da bo malo bolj jasno.